openpype/settings/defaults/project_settings/maya.json

```diff
@@ -30,6 +30,36 @@
             }
         ]
     },
+    "RenderSettings": {
+        "apply_render_settings": true,
+        "default_render_image_folder": "",
```

Comment:
we should set and use those. Currenlty this value is pulled from settings in *Create Render* (but it is not defined there anymore). Be aware that it is used in various places (`collect_render`, deadline plugins, and so on).

Reply:
Thank you, Ondrej, working on that now!